initfiles/componentfiles/configschema/templates/AddKeys.json

```diff
@@ -1,72 +1,129 @@
 {
     "name" : "Add Key",
     "type" : "modification",
-    "inputs" : [
+    "variables" : [
         {
             "name" : "keypair_name",
             "type" : "string",
-            "value" : "mythor"
+            "values" : ["mythor"]
         },
         {
             "name" : "public_key",
             "type" : "string",
-            "value" : "/home/jsmith/.ssh/id_rsa.pub.pem"
+            "values" : ["/home/jsmith/.ssh/id_rsa.pub.pem"]
```

Comment:
are the contents of "values" serving as default values?
Is the content of the values attribute actually inserted into the target env file?
I was expecting place holders, but in this particular case the path seems to be exemplary only.
If its not a placeholder, and we can't use build time vars from the platform to construct these values, let's try to match the out of the box env.xml:
@HOME_DIR@/@RUNTIME_USER@/certificate/public.key.pem
how about "/home/your_user/certificate/plublic.key.pem"

Reply:
This is an example modification template that adds keys. These are example values that would be replaced. I left the template in the PR (actually it has been in the repo for a while) as a discussion point for where these go. Should they be in the repo at all. I feel the decision is outside my realm. So, I expect once the decision is made, we may modify this like what you did above, or maybe just add <place your path here>.

Reply:
Per offline discussion, this file will be removed and documented elsewhere

Reply:
Spoke to Stu, he leans toward leaving it in the repo for now. I am updating the values to show they need to be changed. Richard does not have a preference, so for now just going to leave it in the repo.